components/combobox/private/menu.jsx

```diff
@@ -189,6 +190,7 @@ const Menu = (props) => {
 								}
 					}
 					role="option"
+					tabIndex="0"
```

Comment:
The warning is primarily present so that engineers do not attach tooltips to appear on mouse hover over `div` tags or something like that. Combobox is accessible for screen readers the way it is right now.

Menu listbox item menus shouldn't have `tabIndex`. This will result in focus being changed to them when clicked with the mouse. I think the best direction would be modifying `ToolTip` to not trigger the warning.

Maybe an explicit override prop like`IgnoreIsTriggerTabbable` and then add that to the tooltip usage in the Combobox? What do you think?

Reply:
You are approved to add `IgnoreIsTriggerTabbable` to Tooltip and implment that solution in Combobox.